src/Catel.Core/Catel.Core.Shared/Runtime/Serialization/Attributes/SerializeUsingEnumAsStringAttribute.cs

```diff
@@ -0,0 +1,19 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SerializeUsingParseAndToStringAttribute.cs" company="Catel development team">
+//   Copyright (c) 2008 - 2017 Catel development team. All rights reserved.
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+
+namespace Catel.Runtime.Serialization
+{
+    using System;
+
+    /// <summary>
+    /// Attribute to define that a enum member must be serialized as string.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Property, AllowMultiple = false)]
+    public class SerializeUsingEnumAsStringAttribute : Attribute
```

Comment:
Yes I also think AsString is the better name. But in ISerializerModifier.cs the method  ShouldSerializeEnumMemberUsingToString is Ok? Because you use the toString method?